apps/mull-api/src/app/entities/event.entity.ts

```diff
@@ -76,9 +76,10 @@ export class Event implements IEvent {
   )
   host?: User;
 
-  @ManyToOne(
-    /* istanbul ignore next */ () => Location,
-    /* istanbul ignore next */ (location) => location.events
-  )
+  @Field({ nullable: true })
+  @JoinColumn()
+  @OneToOne(/* istanbul ignore next */ () => Location, (location) => location.event, {
+    cascade: true,
+  })
```

Comment:
This was done since it would simplify some of the queries we needed to change, and since we're currently not taking advantage of this many to one relationship (we're not querying for locations when creating an event to avoid duplicates, etc.)